### ISSUE TITLE
[TW-1747] Due and wait dates should also be inherited

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1771]
_Jens Erat on 2015-12-30T22:37:47Z says:_ 

 If a task A is blocked by another task B in "wait" state, there is some reason I cannot tackle task B right now. But in consequence, task A is indirectly also blocked.

Similar reasoning for the due date: if a blocked task is due before the blocking task, obviously the blocking task must also be finished before that date.

I'm not sure how easy this is to implement, but I'd guess it's rather similar to the urgency inheritance reimplemented already (but I failed at putting together a proof of concept within a reasonable time, not really knowing the inner workings of task warrior).

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-12-30T22:37:47Z
Modified: 2017-01-16T18:30:09Z
```
